Automattic/pocket-casts-android
# Change Snackbar message on existing bookmark

## Description

This PR updates the snackbar message to "bookmark exists" when a bookmark is already added. 

Fixes #  [1288](https://github.com/Automattic/pocket-casts-android/issues/1288)

## Testing Instructions
1. Go to full screen player 
2. Pause playback
3. Add a bookmark at any minute
4. Add bookmark again( Check the Toast message)

## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
